### PR TITLE
Fix/undo-redo on android phones

### PR DESCRIPTION
**Pull request by talevy17**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -50,6 +54,8 @@
 ### :house: Internal
 - `viewer`
   - [#1750](https://github.com/wix-incubator/rich-content/pull/1750) bundle size optimzation: editor translations are excluded from bundle
+- `editor`
```
Comment (shaulgo) [inline CHANGELOG.md:57]: ?

File: packages/plugin-undo-redo/web/src/createToolbar.tsx [hunk 2]
```diff
@@ -41,12 +43,17 @@ const createToolbar: CreatePluginToolbar = ({
           getIcon: () => settings?.toolbars?.icons?.Undo || UndoIcon,
           onClick: e => {
             e.preventDefault();
-            setEditorState(EditorState.undo(getEditorState()));
+            let newEditorState = EditorState.undo(getEditorState());
+            if (isMobile && newEditorState.isInCompositionMode()) {
+              // set inCompositionMode property of editorState to false forces draft to rerender
+              newEditorState = createEditorStateWithoutComposition(newEditorState);
+            }
+            setEditorState(newEditorState);
```
Comment (shaulgo) [inline packages/plugin-undo-redo/web/src/createToolbar.tsx:51]:
```suggestion
         	const newState = utils.undo(getEditorState());
            setEditorState(newState);
```

Comment (shaulgo) [reply packages/plugin-undo-redo/web/src/createToolbar.tsx:51]: let's do something more like this for undo/redo.

Comment (shaulgo) [reply packages/plugin-undo-redo/web/src/createToolbar.tsx:51]: do you need to test for mobile? I think in korean you enter composition mode also in desktop.

**Review by shaulgo — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (talevy17) [reply CHANGELOG.md:57]: Fixed a mistake in changelog I noticed

File: packages/plugin-undo-redo/web/src/createToolbar.tsx (hunk 2 above)
Comment (talevy17) [reply packages/plugin-undo-redo/web/src/createToolbar.tsx:51]: I think checking the composition mode should be enough or at least do no harm, I’ll change

**Review by shaulgo — APPROVED** [review]

**Comment by shaulgo** [comment]
add changelog and merge

**Pull request merged**